coderd/database/migrations/000185_add_user_name.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE users ADD COLUMN name text NOT NULL DEFAULT '';
```

Comment:
Should we default this to `username`?

Reply:
Not sure about this one. I guess that it is similar to "display name" for templates, so rather optionally empty.

Reply:
On the TS side, if it's potentially not set by the user, it might be useful to have as a nullable string?

Reply:
I think that it is more natural for Go to have it just "" (empty string), unless we use *string (nullable) for these cases?

Reply:
It's probably simpler just keep it as-is tbh.